git/types.py

```diff
@@ -102,12 +101,8 @@
     commit-ish. This union type includes :class:`~git.objects.commit.Commit`, but also
     :class:`~git.objects.tag.TagObject`, only **some** of whose instances are
```

Comment:
And here is the second one. It seems like this way of describing it is based on a previous commit where I didn't notice it.